A32NX/layout.json

```diff
@@ -1,619 +1,629 @@
 {
```

Comment:
please dont check this or manifest in anymore, the CI takes care of it, and this jsut creates merge conflicts